test/Deletion.hs

```diff
@@ -36,20 +37,23 @@ tests = describe "Delete" $ do
 
     describe "Proofs" $ do
         it' "Delete proof is verifiable" $ \(k, v, list) -> do
+
             tree        <- AVL.fromList ((k, v) : list) :: StorageMonad M
             (proof,  _) <- AVL.delete' k tree
             (proof1, _) <- AVL.delete' k (AVL.unProof proof)
 
-            let Just hash1 = AVL.rootHash (AVL.fullRehash tree)
+            return $ AVL.checkProof (AVL.rootHash tree) proof1
+
+        it' "dpr Delete proof is replayable" $ \() -> do
```

Comment:
`dpr`? :smile_cat:

Reply:
This is so I was able to rerun the test with `--test-arguments --match=dpr`